README.md

```diff
@@ -102,22 +89,442 @@ docker/waitForDb.py
+| **id**           | String (length of 8 bytes)                              | read-write | No       | Unique identifier for the template.
+| **label**        | String (length of 128 bytes)                            | read-write | Yes      | An user-defined label to facilitate the template's identification.
+| **updated**      | DateTime (with timezone and µs precision) in ISO format | read-only  | No       | Device last update time.
+
+An example template structure:
+
+```json
+{
+  "label": "teste",
+  "attrs": [
+    {
+      "label": "rain",
+      "value_type": "float",
+      "template_id": "1",
+      "id": 1,
+      "static_value": "",
+      "type": "dynamic",
+      "created": "2020-09-16T14:50:09.297163+00:00"
+    },
+    {
+      "label": "mark",
+      "value_type": "string",
+      "template_id": "1",
+      "id": 2,
+      "static_value": "efac",
+      "type": "static",
+      "created": "2020-09-16T14:58:25.905376+00:00"
+    }
+  ],
+  "data_attrs": [
+    {
+      "label": "rain",
+      "value_type": "float",
+      "template_id": "1",
+      "id": 1,
+      "static_value": "",
+      "type": "dynamic",
+      "created": "2020-09-16T14:50:09.297163+00:00"
+    },
+    {
+      "label": "mark",
+      "value_type": "string",
+      "template_id": "1",
+      "id": 2,
+      "static_value": "efac",
+      "type": "static",
+      "created": "2020-09-16T14:58:25.905376+00:00"
+    }
+  ],
+  "id": 1,
+  "config_attrs": [ ],
+  "created": "2020-09-16T14:50:09.292714+00:00"
+}
+```
 
-## API Documentation
+All attributes that are read-write can be used when creating or updating the template. All of them
+are returned when retrieving device data. You might also notice some new attributes:
+- `data_attrs`: stores attributes with the types `dynamic`, `static` and `actuator`.
+- `config_attrs`: stores attributes with the type `meta`. You can only create this type of attribute
+  via API, check its [documentation](https://dojot.github.io/device-manager/apiary_latest.html) for
+  more details.
 
-URL to api documentation for [development](https://dojot.github.io/device-manager/apiary_development.html) and [latest version](https://dojot.github.io/device-manager/apiary_latest.html) of this Device Manager.
\ No newline at end of file
+These two parameters are merged in the `attrs`.
+
+## Events
+
+There are some messages that are published by Device Manager to Kafka. These messages are
+notifications of device management operations, and they can be consumed by any component interested
+in them, such as IoT agents.
+
+For more information on the parameters of the messages, please refer to the [Device Manager concepts
+topic](#concepts).
+
+__NOTE THAT__ all messages reside in Kafka's `dojot.device-manager.device` topic.
+
+The events that are emitted by the Device Manager are:
+
+- `actuate`
+- `create`
+- `remove`
+- `update`
+- `template.update` **deprecated**
+
+### **Event: `actuate`**
```

Comment:
In my tests the event `actuate` doest exist, just the event `configure`

Reply:
`{"event": "configure", "data": {"attrs": {"t": "100"}, "id": "19c216"}, "meta": {"service": "admin", "timestamp": 1600453556258}}`

Reply:
To send a msg to device I use:

```sh
curl -X PUT \
    http://localhost:8000/device/19c216/actuate \
    -H "Authorization: Bearer ${JWT}" \
    -H 'Content-Type:application/json' \
    -d '{"attrs": {"t" : "100"}}'
```